Add demo and share new URL (#356)
cowprotocol/docs: docs/cow-protocol/tutorials/swap-in-safe-app/README.mdx

The easiest way to get started is to follow the steps below.

> ## DEMO

Reviewer: ```suggestion
## Demo
```

Use Sentence Case for headings

Before you start, you can get a feel of the Safe App example. Feel free to skip this step!

The example shows:

> - ✅ How to make a new Safe App.
> - ✅ How to make Swaps, Limit orders and TWAPs.
> - ✅ How to control programatically the swap widget.
> - ✅ How to detect trade parameters (token selection, amounts, etc.).
> - ✅ How to define a dynamic fee.
> - ✅ How to customize the colors, tokens, sounds, etc.

Reviewer: ```suggestion
- How to make a new Safe App
- How to make Swaps, Limit orders and TWAPs
- How to control programatically the swap widget
- How to detect trade parameters (token selection, amounts, etc.)
- How to define a dynamic fee
- How to customize the colors, tokens, sounds, etc
```

As these aren't full sentences, bullet points don't have full stops at the end per style guide. Also, we are generally avoiding the use of emojis excluding top-level navigation items (i.e. CoW Protocol, MEV Blocker etc).

> To use the demo, just follow the step [Add your custom dapp in Safe Wallet](#add-custom-dapp) bellow, but instead of adding the URL http://localhost:9999, use the URL https://safe.widget.cow.fi.

Reviewer: ```suggestion
To use the demo, just follow the instructions in [Add your custom dapp in Safe Wallet](#add-custom-dapp) below, but instead of adding the URL `http://localhost:9999`, use the URL `https://safe.widget.cow.fi`.
```
